src/test/java/TestNGTests/TestDelete.java

```diff
@@ -0,0 +1,35 @@
+package TestNGTests;
+
+import APICalls.DeleteCall;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.io.IOException;
+
+import static APICalls.GetCall.callGet;
+
+
+public class TestDelete extends DeleteCall {
```

Comment:
please check common comments that were added in assertJ and restAssured packages